### DISCUSSION TITLE
Help: Discussion notifications for private repositories?

### DISCUSSION THREAD

**Discussion by sjengle** [community/community#4616] [Discussions]
I used to get notifications for new discussion posts and replies on several private repositories. However, that has stopped working (for at least a week). Using the @ mention in a discussion post also seems to fail to send me a notification.

I am watching "All Activity" for these private repositories. When I visit a post on these discussion boards, I see on the sidebar "You’re receiving notifications because you’re watching this repository."

The notifications are not showing up in email (checked my spam folder). When I go to the notifications page, they are not there. I even tried the filter `is:discussion is:done` to see if I accidentally marked them done, but still nothing.

I have tried changing my watch settings, but that seems to have no effect on whether I see notifications for discussions.

**Comment by Param-Harrison** [comment]
Same here, Can we bubble this up?
Lack of proper notification is killing the feature completely.

**Reply by DavidBergeron-eng** [reply]
Please set up notifications for immediate alarm so I can properly reply
